src/Speckle.Objects/Geometry/Polygon.cs

```diff
@@ -0,0 +1,16 @@
+using Speckle.Sdk.Models;
+
+namespace Speckle.Objects.Geometry;
+
+[SpeckleType("Objects.Geometry.Polygon")]
+public class Polygon : Base
+{
```

Comment:
let's add a required keyword to the boundary, and reconsider the proeprty names based off of other apps that may use polygons